owncloudApp/src/test/java/com/owncloud/android/presentation/viewmodels/settings/SettingsSecurityViewModelTest.kt

```diff
@@ -200,4 +205,30 @@ class SettingsSecurityViewModelTest : ViewModelTest() {
         }
     }
 
+    @Test
+    fun `set pref is security enforced enabled - ok - true`() {
+        every { contextProvider.getBoolean(any()) } returns true
+
+        securityViewModel.isSecurityEnforcedEnabled().apply {
+            assertEquals(true, this)
+        }
```

Comment:
Two things here:
1) Do you think is it necessary to use `apply` here? Being only one line, I think it makes legibility worse. But that's only an opinion, and therefore it's discussable. You can keep it if you like it like that 👍 
2) When checking is true or false, try to use `assertTrue` or `assertFalse` instead of `assertEquals` with a boolean

Reply:
Fixed!